cmd/celestia-appd/cmd/overrides.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"github.com/celestiaorg/celestia-app/pkg/appconsts"
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/spf13/cobra"
+)
+
+// overrideServerConfig applies overrides to the embedded server context's
+// configurations.
+func overrideServerConfig(command *cobra.Command) error {
+	ctx := server.GetServerContextFromCmd(command)
+	ctx.Config.Consensus.TimeoutPropose = appconsts.TimeoutPropose
+	ctx.Config.Consensus.TimeoutCommit = appconsts.TimeoutCommit
+	ctx.Config.Consensus.SkipTimeoutCommit = false
```

Comment:
Isn't this default false anyway? (not that I have anything against being explicit)

Reply:
yeah I believe so. Also, this was just copied from before, but as you said, being explicit isn't the worst thing.